spec/praxis/trait_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe Praxis::Trait do
+
+  subject(:trait) do
+    Praxis::Trait.new do
+      description 'my awesome trait'
+
+      routing do
+        prefix '/:app_name'
+      end
+
+      response :something
+      response :nothing
+
+      params do
+        attribute :app_name, String
+        attribute :order, String,
+          description: "Field to sort by."
+      end
+
+    end
+  end
+
+  context 'describe' do
+    subject(:describe) { trait.describe }
+
+    its([:description]) { should eq('my awesome trait') }
+    its([:responses]) { should eq [[:something,{}], [:nothing,{}]] }
```

Comment:
I'd add an error in there when we do (if that's not that big a deal)...we'll be uncovering possible bugs in apps...